### DISCUSSION TITLE
[Kavita] Smoother Scrolling with Swipe Enabled

### DISCUSSION THREAD

**Discussion by lyingloverr** [Kareadita/Kavita#2582] [Ideas]
### Idea Description

Smoother scrolling for Manga/Comics. See video for example! :D

I believe this will be a good upgrade over the current swiping as it feels more intuitive and is way cleaner overall.

https://github.com/Kareadita/Kavita/assets/108381076/f6fb488c-3ceb-4ae2-b08c-7c56f625fbe2

### Duration of Using Kavita

1 month

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by majora2007** [comment]
I know this is quite old, I just wanted to let you know it's unlikely that this will ever come. The way Kavita coded it's image reader, it only loads one image at a time.
